tests/integration-tests/conftest.py

```diff
@@ -397,6 +426,26 @@ def _write_cluster_config_to_outdir(request, cluster_config):
     return cluster_config_dst
 
 
+def _write_image_config_to_outdir(request, image_config):
+    out_dir = request.config.getoption("output_dir")
+
+    # Sanitize config file name to make it Windows compatible
+    # request.node.nodeid example:
+    # 'dcv/test_dcv.py::test_dcv_configuration[eu-west-1-c5.xlarge-centos7-slurm-8443-0.0.0.0/0-/shared]'
+    test_file, test_name = request.node.nodeid.split("::", 1)
```

Comment:
this code is duplicate, let's create a `_sanitize_filename` method to be used by both methods, or better let's create a generic `_store_config` method to be used to save both cluster and image config, we just need to add some parameters like the destination folder.

Reply:
create a generic method